### PR TITLE
Update revoke-transact-sql.md

### PR DESCRIPTION
**Pull request by alberto-morillo**
This section does not have an example that SQL Server users can refer to.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@alberto-morillo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/t-sql/statements/revoke-transact-sql.md [hunk 1]
```diff
@@ -193,6 +193,31 @@ The use of AS in this statement does not imply the ability to impersonate anothe
...
 |View|[REVOKE Object Permissions &#40;Transact-SQL&#41;](../../t-sql/statements/revoke-object-permissions-transact-sql.md)|  
 |XML Schema Collection|[REVOKE XML Schema Collection Permissions &#40;Transact-SQL&#41;](../../t-sql/statements/revoke-xml-schema-collection-permissions-transact-sql.md)|  
+
+  
+## Examples  
+  
+### A. Grant and revoke
+ **APPLIES TO:**  [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)], [!INCLUDE[ssSDS](../../includes/sssds-md.md)]  
+  
+ The following example creates a demo stored procedure and a contained database user on a user database, then grants execute permissions on the user and finally remove (REVOKE) that permission from the user.
```
Comment (VanMSFT) [inline docs/t-sql/statements/revoke-transact-sql.md:203]:
```suggestion
The following example creates a stored procedure `DemoProc`, and a contained database user on a user database, grants execute permissions on the user, and removes (`REVOKE`) that permission from the user.

```

**Review by VanMSFT — CHANGES_REQUESTED** [review]
Thanks @alberto-morillo! We appreciate these additions. I'm suggesting some changes to the text.

**Comment by PRMerger6** [comment]
@alberto-morillo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by VanMSFT** [comment]
@AndreasWolter - Hey Andreas, do you mind reviewing this new example as well to see if it's appropriate to add to the article?

Additions: https://github.com/MicrosoftDocs/sql-docs/blob/efd9a0d477083e0ade5bed3864c8967923b2fc2d/docs/t-sql/statements/revoke-transact-sql.md#a-grant-and-revoke

Original article: https://docs.microsoft.com/en-us/sql/t-sql/statements/revoke-transact-sql?view=sql-server-ver15

**Comment by alberto-morillo** [comment]
Agree with all changes @VanMSFT. Thank you again for all your appropriate changes.

**Comment by AndreasWolter** [comment]
Thanks
I would like to have a schema name specified for objects, always (and not dbo)
And instead of REVOKE on an individual object simply use Schemas

Thanks
Andreas

-----------------------------------------------
Andreas Wolter | Senior Program Manager (he/him<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmedium.com%2Fgender-inclusivit%2Fwhy-i-put-pronouns-on-my-email-signature-and-linkedin-profile-and-you-should-too-d3dc942c8743&data=04%7C01%7CAndreas.Wolter%40microsoft.com%7C2f11a03e335b4281f77908d89ad83fdd%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637429600865811530%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=JQCb64TawqZ88znj51eRL5FdN59ez%2Bd6yvY7dXCsskU%3D&reserved=0>)
MCM, MCSM | Data Platform Security
Twitter<https://twitter.com/AndreasWolter> | Linkedin<https://www.linkedin.com/in/andreaswolter/>

**Comment by VanMSFT** [comment]
Thanks, @AndreasWolter!

@alberto-morillo - Do you mind implementing Andreas' recommendation into the example? Thanks!

**Comment by alberto-morillo** [comment]
@VanMSFT I don't know how to proceed once I have the new script ready.

**Comment by VanMSFT** [comment]
@alberto-morillo - No problem, here are the steps:

Go to the **Files changed** tab at the top of this PR.

image

Select the three dots (**...**) on the file that you want to change, and then select **Edit file**. You can then proceed with making changes to the file and committing that change to this PR.

image

Commit directly to the current branch:

image

**Comment by PRMerger19** [comment]
@alberto-morillo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by alberto-morillo** [comment]
@VanMSFT Please let me know if you can see the changes I made,

**Comment by PRMerger19** [comment]
@alberto-morillo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by alberto-morillo** [comment]
Please examine the script again. Thank you for all the suggestions.

**Comment by AndreasWolter** [comment]
#sign-off

**Comment by PRMerger10** [comment]
@AndreasWolter: I'm sorry - only the author of this article, @VanMSFT, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by VanMSFT** [comment]
#sign-off

**Comment by PRMerger12** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by alberto-morillo** [comment]
@VanMSFT Good morning! Should I delete now the branch as GitHub is suggesting me to do?

_"Pull request successfully merged and closed
You’re all set — the alberto-morillo:patch-2 branch can be safely deleted.
If you wish, you can also delete this fork of MicrosoftDocs/sql-docs in the settings."_

**Comment by VanMSFT** [comment]
@alberto-morillo Yes, you can delete it.

**Pull request merged**